### DISCUSSION TITLE
Does Simon like ☕?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#939] [Galaxy Trivia] [answered]
Richard, Kevin, and James like coffee... Andrew, Anne, and John don't.

**Comment by YashNuhash** [comment]
Yes, Simon likes coffee. The only people mentioned who don't like coffee are Andrew, Anne, and John. Richard, Kevin, and James all like coffee, and Simon is not mentioned in either group. Therefore, the only possible conclusion is that Simon likes coffee.

Since Simon is not in either the "likes coffee" or "doesn't like coffee" group, the only possible conclusion is that he likes coffee.

**Comment by peterbaccaro** [comment]
The pattern appears to be that those with an odd number of letters in their name like coffee and those with an even number of letters in their name don't.

Therefore as Simon has an odd number of letters in the name this means that he likes coffee.

**Yes, Simon likes coffee**.

**Comment by CelioAmaral** [comment, marked as answer]
Richard, Kevin, and James like coffee... Andrew, Anne, and John don't. Yes, Simon likes coffee?

Richard = 7
Kevin = 5
James = 5

Andrew = 6
Anne = 4
John = 4

Simon = 5

Yes, Simon likes coffee. ; D
